machine/machine_core/machine_virtual.py

```diff
@@ -100,6 +100,12 @@ def stdchannel_redirected(stdchannel: TextIO, dest_filename: str) -> Iterator[No
     <rng model='virtio'>
       <backend model='random'>/dev/urandom</backend>
     </rng>
+    <tpm model='tpm-crb'>
+      <backend type='emulator' version='2.0'>
```

Comment:
Do we need to give a version?

Reply:
No idea... I copied this snippet from one of my own VMs after using c-machines to add TPM to it...